src/main/java/com/degustudios/dotnetformat/NativeCommandRunnerImpl.java

```diff
@@ -0,0 +1,50 @@
+package com.degustudios.dotnetformat;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import ut.com.degustudios.bitbucket.mergechecks.NativeCommandRunner;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.concurrent.Executors;
+
+@Component
+public class NativeCommandRunnerImpl implements NativeCommandRunner {
+
+    private final Logger logger = LoggerFactory.getLogger(NativeCommandRunnerImpl.class);
+
+    private final ProcessBuilder processBuilder = new ProcessBuilder();
```

Comment:
Shouldn't we use new `ProcessBuilder` instance each time we need to build a process?